plugins/flytekit-duckdb/flytekitplugins/duckdb/task.py

```diff
@@ -21,19 +42,35 @@ class DuckDBQuery(PythonInstanceTask):
     def __init__(
         self,
         name: str,
-        query: Union[str, List[str]],
+        query: Optional[Union[str, List[str]]] = None,
```

Comment:
Can you update the docstring to example that `query=None` is useful for runtime queries?